<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>COptionsListEditorGuiComp</class>
 <widget class="QWidget" name="COptionsListEditorGuiComp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>656</width>
    <height>513</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTreeWidget" name="ParamsTree">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
     <property name="columnCount">
      <number>1</number>
     </property>
     <attribute name="headerVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>List</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="ButtonsFrame" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="AddRemoveButtonsFrame" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="AddButton">
           <property name="toolTip">
            <string>Insert new item</string>
           </property>
           <property name="text">
            <string>Add</string>
           </property>
           <property name="icon">
            <iconset resource="../iqtgui/iqtgui.qrc">
             <normaloff>:/Icons/Add.svg</normaloff>:/Icons/Add.svg</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="RemoveButton">
           <property name="toolTip">
            <string>Remove selected item</string>
           </property>
           <property name="text">
            <string>Remove</string>
           </property>
           <property name="icon">
            <iconset resource="../iqtgui/iqtgui.qrc">
             <normaloff>:/Icons/Delete.svg</normaloff>:/Icons/Delete.svg</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>329</width>
          <height>17</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QWidget" name="UpDownButtonsFrame" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QToolButton" name="UpButton">
           <property name="toolTip">
            <string>Move selected item up</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="../iqtgui/iqtgui.qrc">
             <normaloff>:/Icons/Up.svg</normaloff>:/Icons/Up.svg</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="DownButton">
           <property name="toolTip">
            <string>Move selected item down</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="../iqtgui/iqtgui.qrc">
             <normaloff>:/Icons/Down.svg</normaloff>:/Icons/Down.svg</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>ParamsTree</tabstop>
 </tabstops>
 <resources>
  <include location="../iqtgui/iqtgui.qrc"/>
 </resources>
 <connections/>
</ui>
